core/templates/pages/exploration-player-page/learner-experience/tutor-card.directive.ts

```diff
@@ -203,9 +203,15 @@ angular.module('oppia').directive('tutorCard', [
           ctrl.$onInit = function() {
             $scope.arePreviousResponsesShown = false;
             $scope.waitingForOppiaFeedback = false;
-            $scope.isAudioPlaying = false;
             $scope.windowDimensionsService = WindowDimensionsService;
             $scope.isIframed = UrlService.isIframed();
+            $scope.addMarginInMobile = false;
```

Comment:
I already said, in the previous review, not to start variable names with verbs...

Reply:
Fixed